app/konnect/service-catalog/integrations/pagerduty.md

```diff
@@ -0,0 +1,34 @@
+---
+title: PagerDuty Integration
+content-type: reference
+config:
+    discovery_support: true
+    discovery_default: true
+    bindable_entities: "PagerDuty Service"
+    mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The PagerDuty integration allows you to provide a way to alert the service team (via PagerDuty services) as well as provide information on current open incidents to consumers of the service directory. 
+
+For each linked PagerDuty service, a summary should be provided showing current unresolved incidents and the current on-call user.
```

Comment:
Should we instead say something like "...a summary **will** be provided **on the Service Catalog service's details page**, showing current unresolved incidents and the current on-call user."?